### PR TITLE
Updated recipe for TrackGit

### PR DESCRIPTION
**Pull request by Vidrep**
Changed URL to point to HaikuArchives master, which now allows TrackGit to build on 64 bit. Tested as building and working.

### CODE REVIEW

**Comment by Begasus** [comment]
libgit should be defined as liggit >= 0.27 (the 0.26.* fails to build it)

**Comment by Vidrep** [comment]
Added new recipe for libgit2-0.27.5

File: haiku-apps/trackgit/trackgit-1.0.0~git.recipe [hunk 1]
```diff
@@ -2,31 +2,31 @@ SUMMARY="Tracker add-on for git"
 DESCRIPTION="Tracker Addon for Git Version Control System. \
 TrackGit provides user with GUIs for various Git commands. The implementation \
 was done using LibGit2. TrackGit uses the Dynamic population of menu."
-HOMEPAGE="https://github.com/Hrily/TrackGit"
+HOMEPAGE="https://github.com/HaikuArchives/TrackGit"
```
Comment (Begasus) [inline haiku-apps/trackgit/trackgit-1.0.0~git.recipe]: I would prefer the original homepage being set, my 2 cents

Comment (Begasus) [inline haiku-apps/trackgit/trackgit-1.0.0~git.recipe]: I would prefer the original homepage being set, my 2 cents

**Comment by Begasus** [comment]
LGTM

**Comment by Begasus** [comment]
> Suggestion (for lib and devel)
> 
> ```diff
> -	lib:libgit2 >= 0.27.0
> +	lib:libgit2 >= 0.27
> ```
> 
> Latest version (last time I looked was 0.27.5) :) the change to get it working was the latest commit to the source and the change of libgit2 >= 0.27 ) ;)

@fbrosson not sure about this now, as libgit2 $libVersionCompat equals "0" and not "0.27", so maybe that needs to be changed too (the change from 0.26.* to 0.27.* is needed because of baseline introduced into 0.27.0)
Also, I'm still in favour of keeping the HOMEPAGE set to the original source ... any thoughts here?

File: haiku-apps/trackgit/trackgit-1.0.0~git.recipe [hunk 2]
```diff
@@ -2,31 +2,31 @@ SUMMARY="Tracker add-on for git"
...
+ARCHITECTURES="x86_gcc2 x86 x86_64"
 
 PROVIDES="
 	trackgit = $portVersion
 	app:TrackGit = $portVersion
 	"
 REQUIRES="
 	haiku
-	lib:libgit2
+	lib:libgit2 >= 0.27
```
Comment (alaviss) [inline haiku-apps/trackgit/trackgit-1.0.0~git.recipe]:
```suggestion
	lib:libgit2
```

You don't have to specify the version constraint here, as HaikuPorter will derive it from the build environment automatically

**Comment by Vidrep** [comment]
On a clean install of 64 bit, running pkgman to install libgit2, I get this:

Welcome to the Haiku shell.

~> pkgman install libgit2_devel
100% repochecksum-1 [65 bytes]
Validating checksum for Haiku...done.
100% repochecksum-1 [64 bytes]
Validating checksum for HaikuPorts...done.
The following changes will be made:
  in system:
    install package libssh2-1.8.0-3 from repository HaikuPorts
    install package libgit2-0.27.0-1 from repository HaikuPorts
    install package libgit2_devel-0.27.0-1 from repository HaikuPorts
Continue? [yes/no] (yes) : 

Why does pkgman resolve the correct libgit2 version, but the recipe does not?
I have added the dependency for libssh2 to the recipe, and reworked some other things. Recipe is tested as working only as long as libgit2 >= 0.27 is installed beforehand.

**Pull request merged**